src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqQueueNameConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace CoreWCF.Channels
+{
+    public class MsmqQueueNameConverter
+    {
```

Comment:
This should be internal as it's a helper class. Also there's some formatting issues that need tidying up.

Reply:
Ok, I will fixed, but how to test this code for example? I have tried applying the attribute internalVisibleTo, but not work, because need assembly strong signing